Themis/Models/Authentication.swift

```diff
@@ -32,13 +32,21 @@ class Authentication: NSObject {
     func auth(username: String, password: String, rememberMe: Bool = false) async throws {
         let body = AuthBody(username: username, password: password, rememberMe: rememberMe)
         let request = Request(method: .post, path: "/api/authenticate", body: body)
-        _ = try await RESTController.shared.sendRequest(request) { _ in true }
+        do {
+            _ = try await RESTController.shared.sendRequest(request) { _ in true }
+        } catch RESTError.empty {
+            // do nothing because empty response is intended
+        }
```

Comment:
I feel like this should be something like a separate function in the `RESTController` instead of hacking around all of this object-parsing stuff around. This exact code is duplicated a few lines below, which isn't so nice.

Reply:
Ok, added another extra function without return type.